examples/xed-examples-util.c

```diff
@@ -197,7 +197,7 @@ xed_int64_t xed_atoi_general(char* buf, int mul) {
     xed_int64_t b;
 
     char* p = buf;
-    while(*p && isspace(*p))
+    while(*p && isspace((unsigned char)*p))
```

Comment:
would this be better as (int)? That seems like it matches the man pages on win/lin/mac and netbsd